server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java

```diff
@@ -83,7 +83,7 @@ public final User generateAdministrator(Consumer<CreateRequest.Builder>... popul
   public final User generateAdministrator(Organizations.Organization organization, Consumer<CreateRequest.Builder>... populators) {
     String organizationKey = organization.getKey();
     User user = generate(populators);
-    session.wsClient().organizations().addMember(organizationKey, user.getLogin());
+    session.wsClient().organizationsOld().addMember(organizationKey, user.getLogin());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "organizationsOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)